workspaces/liferay-learn-workspace/client-extensions/liferay-learn-dxp-importer/app.sh

```diff
@@ -63,6 +63,40 @@ if [ -z "$SKIP_UPDATE_EXAMPLES" ] ; then
 	send_slack_message "update_examples.sh finished with return code $UPDATE_EXAMPLES_RC. $ERROR_COUNT entries in error log file."
 fi
 
+if [ -z "$SKIP_GENERATE_ZIPS" ] ; then
+	cd $REPO_FOLDER/docs || exit
+
+	for docs_dir_name in $(find . -maxdepth 4 -mindepth 4 -type f -name "contents.rst" -printf "%h\n" )
```

Comment:
@willnewbury  `contents.rst` is a sphinx file, so we shouldn't rely on it being there, since they'll probably delete it after we move away from sphinx

is there a reason we can't just do:

`find $REPO_FOLDER/docs -name \*.zip -type d`

Then we could throw out all the lgoic about language / product / version - we don't need it